src/libs/actions/Report.ts

```diff
@@ -1106,16 +1078,6 @@ function deleteReportComment(reportID, reportAction) {
         optimisticData.push(optimisticParentReportData);
     }
 
-    // Check to see if the report action we are deleting is the first comment on a thread report. In this case, we need to trigger
-    // an update to let the LHN know that the parentReportAction is now deleted.
-    if (ReportUtils.isThreadFirstChat(reportAction, reportID)) {
-        optimisticData.push({
-            onyxMethod: Onyx.METHOD.MERGE,
-            key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
-            value: {updateReportInLHN: true},
```

Comment:
It was added in this [PR](https://github.com/Expensify/App/pull/23514/files). `updateReportInLHN` isn't used anywhere in the codebase. @marcaaron wdyt, is this safe ro remove?

Reply:
It was added as a hack to force an update. To find out if it's needed we will need to look into why we added it in the first place, remove it, test for regressions.